plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcModule.java

```diff
@@ -77,6 +78,11 @@ public void setup(Binder binder)
         binder.bind(CachingJdbcClient.class).in(Scopes.SINGLETON);
         binder.bind(JdbcClient.class).to(Key.get(CachingJdbcClient.class)).in(Scopes.SINGLETON);
 
+        newSetBinder(binder, Procedure.class)
+                .addBinding()
+                .toProvider(FlushJdbcMetadataCacheProcedure.class)
+                .in(Scopes.SINGLETON);
```

Comment:
single line

Reply:
Done and moved to separate PR: https://github.com/trinodb/trino/pull/10265